VERSION

```diff
@@ -0,0 +1 @@
+1.1.1
```

Comment:
Regardless of what we decide to do (required property or delete the property), I think this will be a major version change.

To trigger a major version change, you'll need a commit message that looks like:
```
feat: <commit message>

BREAKING CHANGE: <why it breaks>
```